tests/test_universe.py

```diff
@@ -575,9 +575,9 @@ class PointPadFp32TC(PointPadTB, unittest.TestCase):
 
     def setUp(self):
         self.dtype = 'float32'
-        self.akls = modmesh.SimpleArrayFloat32
-        self.vkls = modmesh.Point3dFp32
-        self.pkls = modmesh.PointPadFp32
+        self.Array = modmesh.SimpleArrayFloat32
```

Comment:
This should be named as `SimpleArray` instead of `Array`.  There may be other array types in the future and `Array` will be ambiguous.